### PR TITLE
Feature market

### PR DESCRIPTION
**Pull request by antonstarasotnikau**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,17 +3,93 @@
...
+        <jackson.version>4.0.0</jackson.version>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-autoconfigure</artifactId>
-            <version>2.7.4</version>
+            <version>${spring-boot.version}</version>
+        </dependency>
```
Comment (AlexeyKatomachin) [inline pom.xml:24]: please, use parent

org.springframework.boot
spring-boot-starter-parent
3.0.0
and remove version from dependencies

File: src/main/java/by/lawaksoft/tradebot/client/ClientNavigation.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package by.lawaksoft.tradebot.client;
+
+import org.springframework.util.StringUtils;
+
+public interface ClientNavigation {
+
+	String OKX_URL = "https://www.okx.com/api/v5";
```
Comment (AlexeyKatomachin) [inline src/main/java/by/lawaksoft/tradebot/client/ClientNavigation.java:7]: please, move url params into properties file

File: src/main/java/by/lawaksoft/tradebot/controller/Navigation.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package by.lawaksoft.tradebot.controller;
+
+public interface Navigation {
+
+	String MARKET = "/market";
```
Comment (AlexeyKatomachin) [inline src/main/java/by/lawaksoft/tradebot/controller/Navigation.java:5]: please, move url params into properties file

File: src/main/java/by/lawaksoft/tradebot/dto/model/market/TickerDto.java [hunk 4]
```diff
@@ -0,0 +1,122 @@
+package by.lawaksoft.tradebot.dto.model.market;
+
+public class TickerDto {
```
Comment (AlexeyKatomachin) [inline src/main/java/by/lawaksoft/tradebot/dto/model/market/TickerDto.java:3]: please, use lombok for dto's

**Review by AlexeyKatomachin — COMMENTED** [review]
please, use lombok

**Review by TorusTredent — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (antonstarasotnikau) [reply pom.xml:31]: Done

File: src/main/java/by/lawaksoft/tradebot/client/ClientNavigation.java (hunk 2 above)
Comment (antonstarasotnikau) [reply src/main/java/by/lawaksoft/tradebot/client/ClientNavigation.java]: Done

File: src/main/java/by/lawaksoft/tradebot/controller/Navigation.java (hunk 3 above)
Comment (antonstarasotnikau) [reply src/main/java/by/lawaksoft/tradebot/controller/Navigation.java]: Done

File: src/main/java/by/lawaksoft/tradebot/dto/model/market/TickerDto.java (hunk 4 above)
Comment (antonstarasotnikau) [reply src/main/java/by/lawaksoft/tradebot/dto/model/market/TickerDto.java:12]: Done

**Review by AlexeyKatomachin — APPROVED** [review]

**Pull request merged**